docs/website/package.json

```
@@ -16,7 +17,7 @@
     "watch:less": "chokidar --initial \"**/*.less\" -c \"npm run build:less\"",
     "watch:ts": "webpack --watch",
     "lint": "concurrently \"npm:lint:js\" \"npm:lint:css\"",
-    "lint:js": "eslint \"src/**/*.{ts,js}\" --max-warnings 3",
+    "lint:js": "eslint \"*.{ts,js}\" \"**/*.{ts,js}\" --max-warnings 3",
```

```suggestion
    "lint:js": "eslint \"**/*.{ts,js}\" --max-warnings 3",
```

 `**` according to glob notation means 0 to any directories so
 `\"*.{ts,js}\"` is included into `\"**/*.{ts,js}\"`